app_server/app/Router.js

```diff
@@ -83,7 +85,6 @@ server.register({
   }
 });
 
-/* Register controllers */
```

Comment:
Hmm on a second look, I think we should leave the `Authenticator` there in other places. `sever.app.authenticator` should be the only exception in `SocketAdapter` as we use `Authenticator` a lot for other static constants. Between, it's a singleton so should be safe.